unconference/LMIC-know-how.md

```diff
@@ -0,0 +1,200 @@
+# Democratising LMIC open science by sharing techniques to successfully engage with global north / HIC funders
+
+## Project description
+**Description**: Collaboratively creating a curriculum and coaching program to ensure that "hidden curriculum" is less hidden, so that funders and potential grantees from different income levels can engage fairly.
+
+**Proposed outcome**: Project proposal to share with global north philanthropy looking to make a difference in the global south
+
+**Value added**: Levelling the playing field with regards to access to resources
+
+
+## Dictionary:
```

Comment:
this whole section loses the line breaks in markdown - you could add additional line breaks between items, turn this into a list, ...

Reply:
Fix for this in https://github.com/marimeireles/disc-unconference-2023-projects/pull/1 :)